addons/mail/models/mail_message.py

```diff
@@ -592,6 +593,9 @@ def create(self, values_list):
             if 'record_name' not in values and 'default_record_name' not in self.env.context:
                 values['record_name'] = self._get_record_name(values)
 
+            if 'internal' not in values and 'default_internal' not in self.env.context and values.get('subtype_id'):
```

Comment:
Maybe the result of `default_get` should be tested, because this will ignore overrides and `ir.default`.